README.md

```diff
@@ -1 +1,137 @@
-# Hello-Microverse-Project
\ No newline at end of file
+# Hello-Microverse-Project
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [🔭 Future Features](#future-features)
+
```

Comment:
- [ ] Just as the previous reviewer said, kindly add all sections from the [README Template](https://github.com/microverseinc/readme-template). 
 - The addition should be both on the header, and respective sections 
 - I've attached the image as needed. please compare it t what you have, and let's make your project very professional 👍 

## View 

![image](https://user-images.githubusercontent.com/50990925/218974191-1a1e3498-a4e2-4bd0-9419-acbb7eccb21b.png)

Reply:
@Shedrack-Sunday Well noted. Kindly note that the previous review was indicated as optional and I did all the items expected of me in the optional section.

Reply:
@Shedrack-Sunday In regard to the PR request. This happened because I changed the branch name. I immediately flagged it to the reviewer then and advised as per the screenshot below
![Screenshot (384)](https://user-images.githubusercontent.com/105988434/218989191-4b673553-4c28-4b6a-b3f8-b28de1f52ecb.png)

Reply:
@Shedrack-Sunday Sorry, Kindly find the correct screenshot below with the previous reviewer's instructions. Also, double checked with my peers as well.
![Screenshot (386)](https://user-images.githubusercontent.com/105988434/218989555-4a3d9512-52d0-4062-8a81-e4fa5b6f7cbe.png)

Reply:
> @Shedrack-Sunday Well noted. Kindly note that the previous review was indicated as optional and I did all the items expected of me in the optional section.

Hello @mirwan918, from his review, this was not optional and that's why I referenced it. 
- Optional comment comes with - [Optional]. Kindly take note of this

## Instruction 
 
![image](https://user-images.githubusercontent.com/50990925/218991781-ce31b273-6bad-48e7-8b9c-4f0a095a3ebf.png)

Reply:
> @Shedrack-Sunday Sorry, Kindly find the correct screenshot below with the previous reviewer's instructions. Also, double checked with my peers as well. ![Screenshot (386)](https://user-images.githubusercontent.com/105988434/218989555-4a3d9512-52d0-4062-8a81-e4fa5b6f7cbe.png)

@mirwan918  I understand. I might have missed that.I  Apologies. But the changes for the README were not an option 😉 
- In case of next time, you can always reopen a closed PR via a button at the bottom of the PR.
- Kindly make the README change, then send for another review 👏

Reply:
@Shedrack-Sunday Well noted and thank you for your swift reply. I will reopen the old PR. Thanks for the clarity on README. I will work on it.

Reply:
You're welcome